packages/store/src/StoreCore.sol

```diff
@@ -436,7 +436,9 @@ library StoreCore {
     memoryPointer += staticLength;
 
     // Append the encoded dynamic length
-    Memory.store(memoryPointer, dynamicDataLength.unwrap());
+    assembly {
+      mstore(memoryPointer, dynamicDataLength)
+    }
```

Comment:
for just mstore I think using assembly directly is clearer, and this method was only used in this 1 place